Breeze.Sharp/EntityAspect.cs

```diff
@@ -931,13 +931,14 @@ internal EntityAspect(IEntity entity)
         var npEntity = GetValue<IEntity>(np);
         // property is already linked up
         if (npEntity != null) {
-          if (npEntity.EntityAspect.IsDetached) {
+          if (npEntity.EntityAspect.IsDetached && np.ForeignKeyProperties.Any()) {
             // need to insure that fk props match
             var fkProps = np.ForeignKeyProperties;
             npEntity.EntityAspect.EntityType = np.EntityType;
             // Set this Entity's fk to match np EntityKey
             // Order.CustomerID = aCustomer.CustomerID
-            npEntity.EntityAspect.EntityKey.Values.ForEach((v, i) => SetDpValue(fkProps[i], v));
+            //commented because set keys to default values
```

Comment:
In https://github.com/enkodellc/blazorboilerplate/blob/development/src/Shared/BlazorBoilerplate.Shared/Dto/Db/UserProfile.cs

ApplicationUser has wrong key as empty Guid